app/src/common/shared/com/igalia/wolvic/ui/widgets/WebXRInterstitialWidget.java

```diff
@@ -103,12 +104,11 @@ private void initializeControllers() {
             addController(DeviceType.PicoNeo3, WebXRInterstitialController.HAND_RIGHT);
         } else if (DeviceType.isHVRBuild()) {
             // TODO identify the specific type of controller
-            int hvrDeviceType = DeviceType.HVR3DoF;
-            if (hvrDeviceType == DeviceType.HVR3DoF) {
-                addController(hvrDeviceType, WebXRInterstitialController.HAND_NONE);
+            if (PlatformActivity.isPositionTrackingSupported()) {
```

Comment:
So the key here is that you are not implementing that method in very single PlatformActivity class we have for the different platforms. The problem again is that there is no PlatformActivity interface common to all of them, so I guess you'd have to go one by one.

Reply:
Good point. I have updated the other implementations of `PlatformActivity`.